packages/lab/src/layout/FlexLayout/FlexLayout.css

```diff
@@ -1,3 +1,16 @@
+.uitk-density-touch {
+  --uitkFlexLayout-gap-density-multiplier: 2;
+}
+.uitk-density-low {
+  --uitkFlexLayout-gap-density-multiplier: 3;
+}
+.uitk-density-medium {
+  --uitkFlexLayout-gap-density-multiplier: 3;
+}
+.uitk-density-high {
+  --uitkFlexLayout-gap-density-multiplier: 3;
+}
+
 /* VARIABLES */
 .uitkFlexLayout {
   --uitkFlexLayout-gap-multiplier: 1;
```

Comment:
we tend to use this --uitkFlexLayout-gap-multiplier as a CSS API, and our own local variables as --flex-layout-gap-multiplier (see Button for example)

so you'll have something like...
.uitk-density-high {
--flex-layout-gap-density-multiplier: 3;
}
etc..
and
.uitkFlexLayout {
--flex-layout-gap-multiplier: var(--uitkFlexLayout-gap-multiplier, 1);
--flex-layout-gap: calc(var(--uitk-size-unit) * var(--flex-layout-gap-multiplier) * var(--uitkFlexLayout-gap-density-multiplier, --flex-layout-gap-density-multiplier));
}

Reply:
makes sense, I'll update those